dafni_cli/models/upload.py

```diff
@@ -34,6 +34,10 @@ def upload_model(
                                    upload a new model.
         json (bool): Whether to print the raw json returned by the DAFNI API
     """
+    if not is_valid_image_file(image_path):
+        click.echo("Invalid image file format")
```

Comment:
Could we also print out the valid file types, perhaps something like
`Wrong definition file type '.json', valid file types are '.tar', '.tar.gz'`